assets/styles.css

```diff
@@ -1 +1,1217 @@
+  font-size: 1.06rem;
+  font-weight: var(--bold-weight);
+  color: var(--primary-color);
+}
+
+#program ul li span {
+  display: inline-block;
+  width: 8.5%;
+}
+
+#program ul li strong {
+  color: var(--primary-color);
+  font-size: 1.1rem;
+  font-weight: var(--black-weight);
+  width: 20%;
+}
+
+.speaker-info p {
+  font-size: 1rem;
+  line-height: 1.4rem;
+}
+
+#footer p {
+  width: 40%;
+  font-size: 1rem;
+  line-height: 1.2rem;
+  font-weight: var(--bold-weight);
+}
+
+#about article p {
+  position: absolute;
+}
+
+#program ul li p {
+  width: 65%;
+}
+
+/* FEATURED SPEAKERS STYLES */
+#speakers {
+  padding: 2rem 0 3rem;
+}
+
+#speakers h2 {
+  color: var(--secondary-color);
+}
+
+#speakers ul {
+  list-style: none;
+  padding: 3rem 0 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+}
+
+#speakers ul li.speaker,
+.hide {
+  display: none;
+}
+
+#speakers ul li.speaker:nth-child(-n+2) {
+  display: flex;
+}
+
+#speakers ul li.speaker.show,
+.btn-less.show {
+  display: flex;
+}
+
+#speakers .speaker {
+  height: 15rem;
+  justify-content: space-between;
+}
+
+#speakers .speaker .speaker-photo {
+  width: 28%;
+  position: relative;
+}
+
+.speaker-photo .squares {
+  height: 4rem;
+  width: 4rem;
+  background-image: url(./bgimgs/pattern.svg);
+  position: relative;
+  top: 0;
+  left: 0;
+}
+
+.speaker-photo img {
+  position: absolute;
+  top: 0.95em;
+  left: 0.95em;
+}
+
+#speakers .speaker .speaker-info {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 65%;
+}
+
+#speakers .speaker-dash {
+  width: 8%;
+  margin: 0.6em 0;
+  border: 1px solid var(--light-gray-color);
+}
+
+.btn-more .arrow-icon::after,
+.btn-less .arrow-icon::after {
+  display: inline-block;
+  content: "";
+  width: 1em;
+  height: 1em;
+  margin-left: 0.5em;
+}
+
+.btn-more .arrow-icon::after {
+  background: url('./icons/arrow-down.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.btn-less .arrow-icon::after {
+  background: url('./icons/arrow-up.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* PARTNERS STYLES */
+#partners {
+  padding: 2rem 0;
+  background-color: var(--secondary-color);
+}
+
+#partners h2 {
+  color: #fff;
+}
+
+.partners-logos {
+  justify-content: space-evenly;
+  margin: 2em 1em;
+}
+
+.partners-logos .partner-logo {
+  display: inline-block;
+  width: 28%;
+  padding: 0.3em;
+}
+
+.show-logo img {
+  max-width: 70%;
+  transition: all 250ms ease-out;
+}
+
+.show-logo img:hover {
+  filter: saturate(2);
+  scale: 1.05;
+}
+
+#partners img {
+  width: 100%;
+  filter: saturate(0);
+  transition: all 200ms ease-in-out;
+}
+
+#footer img {
+  width: 15%;
+  padding: 0.5em;
+}
+
+#partners img:hover {
+  scale: 1.2;
+  filter: saturate(1);
+}
+
+/* FOOTER STYLES */
+#footer {
+  padding: 2rem 0;
+}
+
+#footer .footer-text {
+  display: flex;
+  width: 100%;
+  gap: 1em;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Please note that the footer should contain a logo aligned to the left and text aligned to the right. currently, you are justifying them to the center, kindly fix it .

your design : 
![image](https://user-images.githubusercontent.com/89282221/204111451-9dfe5283-7dd8-4eea-8b4b-911430ba5384.png)

required design 
![image](https://user-images.githubusercontent.com/89282221/204111458-c2687376-a842-4eb7-b1bc-30b88a10e93b.png)


Tip:  

```CSS
#footer .footer-text {
  display: flex;
  width: 100%;
  gap: 1em;
   /* I suggest using justify-around instead of justify-center */
  justify-content: around;
  align-items: center;
}

```

Reply:
Thank you for your kindly feedback, do you think this is ok?
![image](https://user-images.githubusercontent.com/67355479/204113047-e1dfaaf5-7a72-44f8-8531-2473119eee78.png)

Cheers,
Luis